pytorch_lightning/accelerators/accelerator.py

```diff
@@ -213,23 +221,29 @@ def predict(self, args):
         with self.precision_plugin.predict_context(), self.training_type_plugin.predict_context():
             return self.training_type_plugin.predict(*args)
 
-    def training_step_end(self, output):
+    def training_step_end(
+        self, output: Union[torch.Tensor, Dict[str, torch.Tensor], None]
+    ) -> Union[torch.Tensor, Dict[str, torch.Tensor], None]:
         """A hook to do something at the end of the training step
 
         Args:
             output: the output of the training step
         """
         return self.training_type_plugin.training_step_end(output)
 
-    def test_step_end(self, output):
+    def test_step_end(
+        self, output: Union[torch.Tensor, Dict[str, torch.Tensor], None]
+    ) -> Union[torch.Tensor, Dict[str, torch.Tensor], None]:
         """A hook to do something at the end of the test step
 
         Args:
             output: the output of the test step
         """
         return self.training_type_plugin.test_step_end(output)
 
-    def validation_step_end(self, output):
+    def validation_step_end(
+        self, output: Union[torch.Tensor, Dict[str, torch.Tensor], None]
```

Comment:
I think that the recommendation for `Union[..., None]` or `Optional[Union[...]]`

Reply:
I don't get you

Reply:
You think I should replace this? Optional is just an alias and is convenient when having only one other type. I think the current way is fine

Reply:
I am fine with it, just a side comment